.github/workflows/ci.yaml

```diff
@@ -1,6 +1,8 @@
 name: ci
 
-on: [pull_request] # yamllint disable-line rule:truthy
+on:
+  - pull_request
+  - workflow_dispatch
```

Comment:
For testing the yamllint job, I would prefer if we added a `yamllint` target to the `Makefile` and use `make yamllint` in the workflow here, although the `apt-get` part would be trick in the `Makefile` as it would work only on debian based systems.  @vdemeester wdyt?

Reply:
I agree, I went for the quickest but indeed I agree with you. We may want to split this into an « install yamllint » action and a make yamllint 😇

Reply:
Thanks for your comments, will make yamlint available for local testing.

As for workflow_dispatch, it is needed to run tests in my fork before I submit a PR. There is another request for this feature https://github.com/tektoncd/pipeline/issues/8621

Reply:
Let me know if you want me remove workflow_dispatch - due to some security or other reason